app/tasks/bookmark_save_task.py

```
@@ -1,93 +1,231 @@
+                ]
+                
+                for pattern in patterns_to_delete:
+                    # 패턴으로 시작하는 source_id들 삭제
+                    pattern_deleted = await vector_service.delete_documents_by_pattern(
+                        session=session,
+                        user_id=user_id,
+                        source_id_pattern=pattern
+                    )
+                    total_deleted += pattern_deleted
+                    
+            except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)